### DISCUSSION TITLE
Why is it bad to change the blocks for recipes in config? And trying to find the tardis seed recipe config!!

### DISCUSSION THREAD

**Discussion by Servaris1111** [eccentricdevotion/TARDIS#857] [Q&A]
Im struggling to find the tardis seed block config recipe to change the blocks?. Also i am making a modded/plugin server and i want to change the recipe for the tardis to be stupidly hard modded items. If u guys can point me around to why this is bad to change recipe blocks and how i can do it anyway to say a modded block if possible lmk CHEERS!!

**Comment by eccentricdevotion** [comment]
TARDIS seed block recipes have never been configurable, apart from selecting the wall blocks from a pre-defined list.
In the latest version of the plugin, TARDIS item recipes are no longer configurable. In previous versions you could change the recipes in _recipes.yml_ - though I'm pretty sure adding modded blocks and items will not work as they are not part of the Bukkit/Spigot API.

**Reply by eccentricdevotion** [reply]
You could try editing the `tardis_blocks` list in _blocks.yml_, but like I said above Spigot has no idea what modded blocks are.
